style.css

```diff
@@ -248,39 +249,62 @@
   font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 16px;
-  line-height: 24px;
+  font-size: 1rem;
+  line-height: 1.5rem;
   text-align: center;
   letter-spacing: 0.03em;
   color: #fff;
   border: none;
 }
 
-.various-projects {
+.various-projects-1,
```

Comment:
- [ ] Please add a border to this project. It's a project requirement to follow the template in Figma.

Reply:
please explain further